src/gsy_e/models/strategy/storage.py

```diff
@@ -67,75 +69,103 @@ def serialize(self):
-            energy_rate_decrease_per_update=energy_rate_decrease_per_update)
+            energy_rate_decrease_per_update=energy_rate_decrease_per_update,
+        )
 
         if isinstance(update_interval, int):
             update_interval = duration(minutes=update_interval)
 
         super().__init__()
 
         self.offer_update = TemplateStrategyOfferUpdater(
-            initial_rate=initial_selling_rate, final_rate=final_selling_rate,
+            initial_rate=initial_selling_rate,
+            final_rate=final_selling_rate,
             fit_to_limit=fit_to_limit,
             energy_rate_change_per_update=energy_rate_decrease_per_update,
-            update_interval=update_interval)
+            update_interval=update_interval,
+        )
         for time_slot in self.offer_update.initial_rate_profile_buffer.keys():
             StorageValidator.validate(
                 initial_selling_rate=self.offer_update.initial_rate_profile_buffer[time_slot],
                 final_selling_rate=get_from_profile_same_weekday_and_time(
-                    self.offer_update.final_rate_profile_buffer, time_slot))
+                    self.offer_update.final_rate_profile_buffer, time_slot
+                ),
+            )
         self.bid_update = TemplateStrategyBidUpdater(
-            initial_rate=initial_buying_rate, final_rate=final_buying_rate,
+            initial_rate=initial_buying_rate,
+            final_rate=final_buying_rate,
             fit_to_limit=fit_to_limit,
             energy_rate_change_per_update=energy_rate_increase_per_update,
-            update_interval=update_interval, rate_limit_object=min
+            update_interval=update_interval,
+            rate_limit_object=min,
         )
         for time_slot in self.bid_update.initial_rate_profile_buffer.keys():
             StorageValidator.validate(
                 initial_buying_rate=self.bid_update.initial_rate_profile_buffer[time_slot],
                 final_buying_rate=get_from_profile_same_weekday_and_time(
-                    self.bid_update.final_rate_profile_buffer, time_slot))
+                    self.bid_update.final_rate_profile_buffer, time_slot
+                ),
+            )
         self._state = StorageState(
-            initial_soc=initial_soc, initial_energy_origin=initial_energy_origin,
-            capacity=battery_capacity_kWh, max_abs_battery_power_kW=max_abs_battery_power_kW,
-            min_allowed_soc=min_allowed_soc)
+            initial_soc=initial_soc,
+            initial_energy_origin=initial_energy_origin,
+            capacity=battery_capacity_kWh,
+            max_abs_battery_power_kW=max_abs_battery_power_kW,
+            min_allowed_soc=min_allowed_soc,
+            losses=losses,
+        )
         self.cap_price_strategy = cap_price_strategy
         self.balancing_energy_ratio = BalancingRatio(*balancing_energy_ratio)
 
+    @staticmethod
+    def deserialize_args(constructor_args: Dict) -> Dict:
```

Comment:
I think that this method only needs to be implemented if we want to involve gsy-web. Since we do not this could be completely omitted. Having said that, no problem keeping it in case we need it in the future.

Reply:
OK, let me remove it here, only to explicitly add it once/if we add the losses to the FE.